src/controllers/organization.controller.ts

```diff
@@ -1,14 +1,22 @@
 import OrganizationService from '@/services/organization.service';
 import { NextFunction, Request, Response } from 'express';
+import { start } from 'repl';
+import { HttpException } from '@exceptions/HttpException';
 
 class OrganizationController {
   public organizationService = new OrganizationService();
 
   public getOrganizationData = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
     try {
-      const orgId = parseInt(req.params.id);
-      const orgData = await this.organizationService.getOrganizationData(orgId);
+      const orgId = req.params.id;
 
+      if (!req.query.start_date || !req.query.end_date) {
+        throw new HttpException(400, 'Please pass start and end date parameters.');
+      }
+
+      const start_date = req.query.start_date.toString();
+      const end_date = req.query.end_date.toString();
+      const orgData = await this.organizationService.getOrganizationData(orgId, start_date, end_date);
```

Comment:
Use camel case for consistency. 
I used snake case in the `interface.ts` files just to match the database but should've probably used camel case there too